packages/api-clients/open-api/delegationApi.yml

```diff
@@ -118,19 +118,20 @@ paths:
               schema:
                 $ref: "#/components/schemas/Problem"
   /delegations/{delegationId}:
+    parameters:
```

Comment:
Shouldn't we add the `CorrelationIdHeader` too?

Reply:
Yes, we should! Apparently, it's missing everywhere in this spec, I added it in 250733766